packages/authenticator/amplify_authenticator/lib/src/l10n/message_resolver.dart

```diff
@@ -7,13 +7,15 @@ import 'package:flutter/material.dart';
 
 enum MessageResolverKeyType {
   codeSent,
+  copySucceeded,
+  copyFailed,
 }
 
 class MessageResolverKey {
-  const MessageResolverKey._(this.type, this.destination);
+  const MessageResolverKey(this.type, this.destination);
```

Comment:
This should be private. These classes should really be enums but they were from a different time.